pkg/skuba/cloud/cloud.go

```diff
@@ -25,9 +25,9 @@ import (
 	"github.com/SUSE/skuba/pkg/skuba"
 )
 
+// HasCloudIntegration checks if the cluster has a cloud integration
 func HasCloudIntegration() bool {
 	if _, err := os.Stat(skuba.OpenstackCloudConfFile()); err == nil {
-		os.Chmod(skuba.OpenstackCloudConfFile(), 0400)
```

Comment:
If `read-only` is issue then please change it to `read-write` `0600` for `user`. Restricted privileges for files are always good in case when you store credentials in it.

Reply:
Ok, after further discussions with @mjura we decided we will update skuba to write the `.template` file with 0600 permissions and then add a failing check on the `.conf` file if it's permissions are too permissive like `ssh` does.